tox.ini

```diff
@@ -25,72 +25,68 @@ passenv =
 commands =
     py.test --cov=scrapy --cov-report= {posargs:scrapy tests}
 
-[testenv:trusty]
+[testenv:py27-pinned-versions]
 basepython = python2.7
 deps =
-    pyOpenSSL==0.13
-    lxml==3.3.3
-    Twisted==13.2.0
-    boto==2.20.1
-    Pillow==2.3.0
-    cssselect==0.9.1
-    zope.interface==4.0.5
-    -rtests/requirements-py2.txt
-
-[testenv:jessie]
-# https://packages.debian.org/en/jessie/python/
-# https://packages.debian.org/en/jessie/zope/
-basepython = python2.7
-deps =
-    cryptography==0.6.1
-    pyOpenSSL==0.14
-    lxml==3.4.0
-    Twisted==14.0.2
-    boto==2.34.0
-    Pillow==2.6.1
+    -ctests/constraints.txt
     cssselect==0.9.1
-    zope.interface==4.1.1
+    lxml==3.5.0
+    parsel==1.5.0
+    PyDispatcher==2.0.5
+    pyOpenSSL==16.2.0
+    queuelib==1.4.2
+    service_identity==16.0.0
+    six==1.10.0
+    Twisted==16.0.0
+    w3lib==1.17.0
     -rtests/requirements-py2.txt
-# Not used directly but allows boto GCE plugins to load.
-# https://github.com/GoogleCloudPlatform/compute-image-packages/issues/262
-    google-compute-engine==2.8.12
-
-[testenv:trunk]
-basepython = python2.7
-commands =
-    pip install -U https://github.com/scrapy/w3lib/archive/master.zip#egg=w3lib
-    pip install -U https://github.com/scrapy/queuelib/archive/master.zip#egg=queuelib
-    py.test --cov=scrapy --cov-report= {posargs:scrapy tests}
+    # Extras
+    Pillow
+    botocore
 
 [testenv:pypy]
 basepython = pypy
 commands =
     py.test {posargs:scrapy tests}
 
-[testenv:py34]
-basepython = python3.4
+[testenv:py35]
+basepython = python3.5
 deps =
     -ctests/constraints.txt
     -rrequirements-py3.txt
     # Extras
     Pillow
     -rtests/requirements-py3.txt
 
-[testenv:py35]
+[testenv:py35-pinned-versions]
 basepython = python3.5
-deps = {[testenv:py34]deps}
+deps =
+    -ctests/constraints.txt
+    cssselect==0.9.1
+    lxml==3.5.0
+    parsel==1.5.0
+    PyDispatcher==2.0.5
+    pyOpenSSL==16.2.0
+    queuelib==1.4.2
+    service_identity==16.0.0
+    six==1.10.0
+    Twisted==17.9.0
+    w3lib==1.17.0
+    -rtests/requirements-py3.txt
+    # Extras
+    Pillow
```

Comment:
Comparing to the Python 2 pinned environment, I notice botocore missing here. Is this on purpose?